File: problems/doughnut/doughnut.c
#include "stdio.h"

int main()
{
    int t;
    int c,k,w;
    
    scanf("%d", &t);
    while(t--)
    {
        scanf("%d %d %d", &c, &k, &w);
        if(c*w <= k)
            printf("yes\n");
        else printf("no\n");
    }
    return 0;
}

File: problems/doughnut/doughnut.hs
import Control.Monad

enoughDoughnut :: Int -> Int -> Int -> String
enoughDoughnut c k w
  | c*w <= k = "yes"
  | otherwise = "no"


main = do
  line <- getLine
  let t = read line :: Int
  forM [1..t] (\_ -> do
                  line <- getLine
                  let c:k:w:_ = map (\x -> read x :: Int) (words line)
                  putStrLn $ enoughDoughnut c k w)
